mayastor/tests/replica_timeout.rs

```diff
@@ -1,23 +1,33 @@
-#![allow(unused_assignments)]
-
-use common::{bdev_io, compose::Builder, MayastorTest};
+use common::{compose::Builder, MayastorTest};
 use mayastor::{
-    bdev::{nexus_create, nexus_lookup},
+    bdev::{nexus_create, nexus_lookup, NexusStatus},
     core::MayastorCliArgs,
+    subsys::{Config, NvmeBdevOpts},
 };
-use rpc::mayastor::{BdevShareRequest, BdevUri, Null};
+use rpc::mayastor::{BdevShareRequest, BdevUri, Null, ShareProtocolNexus};
+use std::process::{Command, Stdio};
 use tokio::time::Duration;
 
 pub mod common;
 static NXNAME: &str = "nexus";
 
-#[ignore]
 #[tokio::test]
 async fn replica_stop_cont() {
+    // Use a shorter timeouts than the default to reduce test runtime
+    Config::get_or_init(|| Config {
+        nvme_bdev_opts: NvmeBdevOpts {
+            action_on_timeout: 2,
```

Comment:
Should we be explicitly setting action_on_timeout?
The default is already 2 (== SPDK_BDEV_NVME_TIMEOUT_ACTION_ABORT) and maybe we should be testing the default behaviour. Otherwise we should comment what 2 means.

Reply:
Good points, I don't remember why I explicitly set it to the (current) default. I'll drop it.